lib/pa_ess/utilities.ex

```diff
@@ -563,6 +567,11 @@ defmodule PaEss.Utilities do
     String.to_integer(str)
   end
 
+  @spec prediction_four_cars?(Predictions.Prediction.t()) :: boolean()
+  def prediction_four_cars?(prediction) do
+    prediction.route_id == "Red" and match?([_, _, _, _], prediction.multi_carriage_details)
```

Comment:
nit: I think I like the way we do this in `Message.Predictions.to_multi_line/1` better.

```elixir
def prediction_four_cars?(%Prediction{route_id: "Red", multi_carriage_details: [_, _, _, _]}), do: true
def prediction_four_cars?(_prediction), do: false
```